packages/admin/cms-admin/src/dam/DataGrid/selection/DamMoreActions.tsx

```diff
@@ -106,3 +111,23 @@ const StyledDivider = styled(Divider)`
         margin: 8px 10px;
     }
 `;
+
+interface NumberSelectedChipProps {
+    num: number;
+}
+
+const NumberSelectedChip = ({ num }: NumberSelectedChipProps) => {
+    return <StyledChip label={num} />;
+};
+
+const StyledChip = styled(Chip)`
```

Comment:
Why do you use a Chip component as base? It seems that you're overriding all styles anyway.